modules/ROOT/pages/flow-component.adoc

```diff
@@ -45,13 +45,11 @@ up business events. You can also set up error handling for a component
 
 If you set the initial state to Stopped (`initialState="stopped"`), you need to use xref:runtime-manager::flow-management.adoc[Runtime Manager] to activate the flow, or you can simply reset the Flow configuration to `Started`. Note that the console
 output for this state looks something like this: `Flow flow_component_ex has not been started (initial state = 'stopped')`
-| Max Concurrency (`maxConcurrency`) a| Optional. The default is limited only by the availability of threads. The container thread pool determines the maximum level of parallelism that the flow can use to optimize the performance when processing messages.
+| Max Concurrency (`maxConcurrency`) a| Optional. The default is limited only by the availability of threads. The container thread pool determines the maximum level of parallelism that the flow can use to optimize the performance when processing messages. You can use Max Concurrency to prevent the flow from accepting new messages while the configured number of maximum request is being processed.
```

Comment:
I know that you revised only the dark green portions. However, this definition of Max Concurrency doesn't explicitly tell me what Max Concurrency does.
"The default is limited only by the availability of threads." -- Isn't a default always a particular value? For example, in just about every piece of software that I've used where a field has a default value, that field always has the same default value. If a field has a default of 3, that is always its default value. Are you saying that the "default" for this field is always different? If not, what does this sentence mean?
"The container thread pool determines the maximum level of parallelism..." -- How? Do you mean that the number of the threads in the container pool sets the maximum number of messages that can be processes at the same time in a flow? If that is true, why am I allowed to set a value in this field?
"to optimize the performance when processing messages" -- Optimize in what way? And I don't see the connection between this phrase and the earlier part of this sentence.